index.bs

```diff
@@ -459,6 +464,9 @@ This Section defines a number of new set of <a>Constrainable Properties</a> for
   <dt><dfn dict-member for="MediaTrackCapabilities"><code>exposureCompensation</code></dfn></dt>
   <dd>This reflects the supported range of <a>exposure compensation</a>. The supported range can be, and usually is, centered around 0 EV.</dd>
 
+  <dt><dfn dict-member for="MediaTrackCapabilities"><code>exposureTime</code></dfn></dt>
+  <dd>This reflects the supported range of <a>exposure time</a>.</dd>
```

Comment:
For completion, could you add here something like `Values are numeric. Increasing values indicate increasing exposure time` to mimic the other entries?

Reply:
Done